modules/publicize/publicize-jetpack.php

```diff
@@ -442,6 +444,25 @@ function save_publicized( $post_ID, $post, $update ) {
 		}
 	}
 
+	function set_post_flags( $flags, $post ) {
+		$flags[ 'publicize_post' ] = false;
+		if ( ! $this->post_type_is_publicizeable( $post->post_type ) ) {
+			return $flags;
+		}
+
+		if ( ! apply_filters( 'publicize_should_publicize_published_post', true, $post ) ) {
```

Comment:
You know the drill :)

`/** This filter is already documented in modules/publicize/publicize-jetpack.php */`